src/tests/Loader/classloader/StaticVirtualMethods/RegressionTests/GitHub_70385.cs.template

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+interface IBase<T>
+{
+    public static abstract void Method();
+}
+
+interface IDerived<T> : IBase<T>
+{
+    public static void IBase<T>.Method()
```

Comment:
Thanks for your feedback, I have removed the public modifier from both interfaces in the 2nd commit. Please note that the C# code is actually not used for building the test - that's why it has the 'template' extension - because we don't yet have a Roslyn capable of compiling this code, the actual test source code is written in IL.